src/services/user/model/user.schema.js

```diff
@@ -97,6 +97,8 @@ const userSchema = new Schema(
 		ldapId: { type: String }, // UUID to identify during the sync
 		previousExternalId: { type: String }, // the former ldapId after oauth migrated user
 
+		lastSyncedAt: { type: Date }, // timestamp of the last user data sync
```

Comment:
comment sais timestamp, but we don't store timestamp (integer counting miliseconds form 1970) but date format...